extensions/local-switch/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "local-switch",
+  "title": "Local Switch",
```

Comment:
what about renaming the extension to `Host Switch`? There doesn't seem to be anything inherent to the extension or the command that would limit it to switching _from_ localhost to a remote host.

Reply:
Agree that makes sense, renamed 👍